config/config.exs

```diff
@@ -0,0 +1,4 @@
+import Config
+
+config :novu,
+  domain: "https://api.novu.app"
```

Comment:
```suggestion
  domain: "https://api.novu.co"
```

This one was actually a mistake on the docs, we fixed it a couple of days back 😅

Reply:
Thanks @scopsy 🎉 Updated